pages/common/autojump.md

```diff
@@ -1,12 +1,13 @@
 # autojump
 
-> Automatically jump to the directory you visited the most. Aliases like j or jc can also be used for fewer typing.
+> Quickly jump among the directories you visit the most.
+> Aliases like j or jc are provided for even less typing.
 
 - Jump to a directory that contains the given pattern.
 
 `j {{pattern}}`
 
-- Jump to a sub-directory of current directory that contains the given pattern.
+- Jump to a sub-directory (child) of the current directory that contains the given pattern.
```

Comment:
I don't understand why you added (child) here. Sub-directory is a common, very simple for understanding term.